### PR TITLE
Update msw official build tools; add support for VS 2017, 2019

### PR DESCRIPTION
**Pull request by doscott**
Did a complete build of all libraries and created packages using these. I didn't see any problems.

### CODE REVIEW

File: build/tools/msvs/build2010.bat [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
-cd c:\wxWidgets-%1.%2.%3\build\msw
+cd ..\..\msw
 
 rem ================ VS 2010 Official Build ===============
 rem
-rem Copy this file to the \wxwidgets-x.y.z\build\msw folder
-rem
-rem Open a Windows SDK V7.1 command prompt.
+em Open a Windows SDK V7.1 command prompt.
```
Comment (vadz) [inline build/tools/msvs/build2010.bat:11]: Typo?

```suggestion
rem Open a Windows SDK V7.1 command prompt.
```

**Review by vadz — COMMENTED** [review]
Thanks! No real comments, just an apparent typo -- but I can fix this when merging, if you'd like.

File: build/tools/msvs/build2010.bat (hunk 1 above)
Comment (doscott) [reply build/tools/msvs/build2010.bat:11]: Go ahead and fix it.

**Comment by vadz** [comment]
This was merged as 069f2952f752fa79c5a9d7cf4217603020cb76fd

**Pull request closed without merging**